frontend/src/scenes/setup/UpdateEmailPreferences.js

```diff
@@ -10,18 +11,20 @@ export function UpdateEmailPreferences() {
 
     return (
         <div>
-            <label>
-                <input
-                    type="checkbox"
-                    onChange={e => {
-                        // TODO: refactor so we arent re rendering multiple times
-                        userUpdateRequest({ user: { email_opt_in: !user.email_opt_in } })
-                        setSaved(true)
-                    }}
-                    defaultChecked={user.email_opt_in}
-                />
-                &nbsp;Tick this box to receive security and feature updates via email. You can easily unsubscribe at any
-                time.
+            <Switch
+                onChange={checked => {
+                    // TODO: refactor so we arent re rendering multiple times
```

Comment:
I tend not to be a super big fan of TODO comments, as they never tend to get done. Either we do it in the PR, raise an issue or just decide it's not important enough.

Reply:
I agree. However, this was done in https://github.com/PostHog/posthog/blob/4afcde50b3ab477c83bff21d1f00ae9cd72e56f3/frontend/src/scenes/setup/UpdateEmailPreferences.js#L17

I have no idea what this means.

Do you have any insight on this or do you want me to ping the original author?

Reply:
Ah I missed that, feel free to just delete it

Reply:
Done!